### PR TITLE
Remove content negotiation

### PR DESCRIPTION
**Pull request by felipeerias**
This PR removes content negotiation from the spec.

It is more efficient to use content negotiation on the server to select which bundle to send, rather than including several variants in the bundle and having that negotiation happen on the client's side.

Issue: https://github.com/wpack-wg/bundled-responses/issues/2

Adapted from https://github.com/WICG/webpackage/pull/618

### CODE REVIEW

**Review by hayatoito — APPROVED** [review]
Thanks for making a PR!

Per the discussion https://github.com/wpack-wg/bundled-responses/issues/2, the chromium side is okay to remove content negotiation.

We probably have to change parser's behavior by checking a bundle's version. That should not be a significant issue for us. 

We might want to wait for @jyasskin 's approval before merge.

**Comment by jyasskin** [comment]
Now that the working group has adopted the document, we actually need them to approve significant changes like this one, not me. We can check on this one during the IETF meeting [on Monday](https://datatracker.ietf.org/meeting/111/agenda/#2021-07-26-110000).

**Pull request merged**
